data/_ui/tooltips.txt

```diff
@@ -1225,7 +1225,7 @@ tip "Fleet: Fight my target"
 	`Command your entire fleet to focus fire on the selected target. If only some escorts are selected, only sends the command to those escorts. This command can also be sent by right clicking on a hostile ship.`
 
 tip "Fleet: Gather around me"
-	`Command your fleet to gather around your flagship. If only some escorts are selected, only sends the command to those escorts. You can also command escorts to gather around another ship by right clicking on a friendly ship.`
+	`Command your fleet to gather around your flagship. If only some escorts are selected, only sends the command to those escorts. You can also command escorts to gather around another ship by right clicking on a friendly ship. Command ships to change formation when used in combination with shift.`
```

Comment:
This could be more standardized or at least put in a separate paragraph (look at other commands that use shift).